core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -382,6 +402,29 @@ impl SchedulerController {
             );
         }
     }
+
+    /// Calculate prioritization for a transaction.
+    fn calculate_prioritization(
+        fee_structure: &FeeStructure,
+        include_loaded_account_data_size_in_fee: bool,
+        message: &SanitizedMessage,
+        budget_limits: &FeeBudgetLimits,
+        cost: u64,
+    ) -> u64 {
+        let total_fee = fee_structure.calculate_fee(
```

Comment:
nit: I think calling this var just `fee` is fine, because there's no other _`*_fee`_ s other than this `total_fee`.